recipe/activate.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+set "PYEPICS_LIBCA=%EPICS_BASE%\bin\%EPICS_HOST_ARCH%\ca.dll"
```

Comment:
It is not clear to me that these envs will be defined? On the other hand I am not a windows user and am going on pattern matching so please disregard if tihs is OK.

Reply:
`EPICS_BASE` and `EPICS_HOST_ARCH` variables are defined in the the epics-base activate script: https://github.com/conda-forge/epics-base-feedstock/blob/master/recipe/pre-build.py#L17
The pyepics_activate.bat script has to be run after epics-base_activate.bat, which is the case as conda execute activate scripts following alphabetical order.

Reply:
fair, but that seems a bit brittle.  Might be better to eventually prefix these with integers, but no reason to hold this PR up over that.

Reply:
I will go ahead and mark this as resolved as the tests are passing and the current solution works. As @tacaswell mentioned we can address the prefix via another PR.